packages/app/src/app/provider.service.ts

```diff
@@ -37,6 +38,7 @@ export class ProviderService {
         } else {
           return new OpenFeatureLaunchDarklyProvider({
             sdkKey,
+            logger: new OpenFeatureLogger(`LaunchDarkly Provider`),
```

Comment:
I guess you like the idea to add loggers to the constructors?

Reply:
It was just a quick way to remove the console logs.

Reply:
But it seems to work fine.